components/automate-ui/src/app/modules/desktop/desktop-detail/desktop-detail.component.ts

```diff
@@ -1,12 +1,16 @@
 import { Component, OnInit, OnDestroy, Input, Output, EventEmitter } from '@angular/core';
 import { Desktop, DailyNodeRunsStatus } from 'app/entities/desktop/desktop.model';
-import { takeUntil } from 'rxjs/operators';
+import { takeUntil, finalize } from 'rxjs/operators';
+import * as moment from 'moment/moment';
+import { saveAs } from 'file-saver';
 import { DateTime } from 'app/helpers/datetime/datetime';
 import { Store } from '@ngrx/store';
 import { NgrxStateAtom } from 'app/ngrx.reducers';
 import { Subject } from 'rxjs';
 import { GetDailyNodeRunsStatusTimeSeries } from 'app/entities/desktop/desktop.actions';
 import { getDailyNodeRuns } from 'app/entities/desktop/desktop.selectors';
+import { NodeRunsService } from '../../../services/node-details/node-runs.service';
+import { RunHistoryStore } from '../../../services/run-history-store/run-history.store';
```

Comment:
Someone else will need to confirm but as I understand it, we should avoid the `../..`notation and instead start at `app` when necessary.